build/deploy/generate-deploy-examples.sh

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env bash
+
+SCRIPT_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)
+MANIFEST_FILE=../../deploy/examples/multus-validation.yaml
+
+
+
+run_gen() {
+  if [[ ${SKIP_GEN} = true ]]; then
+    echo "SKIP_GEN is set, skipping manifest  generation"
+  else
+      OUT_FILE="${SCRIPT_ROOT}/${MANIFEST_FILE}"
+      TMP_FILE="$(mktemp)"
+
+ START_MARKER='# THE BELOW HELP TEXT IS AUTO GENERATED BY `make docs`$'
+ END_MARKER='# THE ABOVE HELP TEXT IS AUTO GENERATED BY `make docs`$'
+
+
+
+	  go run ${SCRIPT_ROOT}/cmd/rook/*.go multus validation run --help | awk '{print "#" $0}' > ${TMP_FILE}
+
+	sed -e "$START_MARKER/,/$END_MARKER/{ $START_MARKER/{ p; r $TMP_FILE ; /$END_MARKER/p;d}" -I ".orig" ${OUT_FILE}
```

Comment:
Did you come across this? https://superuser.com/a/440057

It looks like it could be necessary to do this on two lines instead of one with posix sed.

Reply:
@BlaineEXE  wote:
> Did you come across this? https://superuser.com/a/440057

yes, this is exactly what guided me :-) 


> It looks like it could be necessary to do this on two lines instead of one with posix sed.

I am afraid I don't fully understand yet what this means .... 😬